src/test/java/seedu/address/model/person/PersonTest.java

```diff
@@ -62,7 +62,7 @@ public void isSamePerson() {
     @Test
     public void equals() {
         // same values -> returns true
-        Person aliceCopy = new PersonBuilder(ALICE).build();
+        Person aliceCopy = new PersonBuilder(ALICE).withFaculty("SOC").build();
```

Comment:
withFaculty("SOC") not needed, since the Person object ALICE has already been created with Faculty field